kitsune/users/migrations/0009_Change_Twitter_link_to_Twitter_handle.py

```diff
@@ -21,8 +21,9 @@ def forwards(self, orm):
 
 
     def backwards(self, orm):
-        """Add twitter.com/ URL before the the Twitter username"""
-        for profile in orm.Profile.objects.exclude(twitter=None):
+        """Add twitter.com/ URL before the the Twitter username."""
+        """Should no add link to the Null field and the field where there is no entry"""
```

Comment:
You have two docstrings here. Either they should be combined, if you meant to have a longer doc string, or you should turn the second one into a comment (like `# Should not add link...`). Docstrings `"""like this"""` should not be used for comments.